docs/ubuntu1604.md

```diff
@@ -0,0 +1,22 @@
+Ubuntu 16.04 prep
+-----------------
+
+The following commands are one possible way to quickly prepare an instance of
+Ubuntu 16.04 for Uno.
+
+```bash
+#install stuff
+sudo apt install --assume-yes openjdk-8-jdk maven git openssh-server wget
```

Comment:
Not a good idea to recommend `--assume-yes` with documentation/instructions. Better to let users respond interactively.